README.md

```diff
@@ -94,6 +94,16 @@ Putting it all together, it is 6 variant test cases.
 - Applying the `Web variant` to a `Standard Web project`
 - Applying the `Web variant` to a `Custom Web project`
 
+### Release
+
+Set the `HEX_API_KEY` as a Github secret (skip this step if it has been done)
+
+The release process is following the [Git flow](https://nimblehq.co/compass/development/version-control/#releases-).
+
+Once a `release/<version number>` is created, to publish the new version to Hex.pm, the version number in the `mix.ex` file needs to be updated on the release branch before merging.
+
+Once the release branch is merged into the master, Github Action will automatically publish the template to https://hex.pm/packages/nimble_phx_gen_template.
```

Comment:
Small clarification:

```suggestion
Once the release branch is merged into the `master` branch, Github Action will automatically publish the template to https://hex.pm/packages/nimble_phx_gen_template.
```

Reply:
Fixed in https://github.com/nimblehq/elixir-templates/pull/63/commits/b6cc5fc1aaf57f9bb3d280bbc73ce105c357b8bf @olivierobert